stackslib/src/chainstate/stacks/boot/signers_voting_tests.rs

```diff
@@ -350,7 +359,7 @@ fn vote_for_aggregate_public_key_in_last_block() {
     );
 
     // second vote should fail with duplicate vote error
-    let tx2 = &receipts[receipts.len() - 1];
+    let tx2 = &receipts[receipts.len() - 2];
```

Comment:
Also better to use explicit names for the tx instead of `tx2`. I'd have to read too much into the test just to find out what tx this is testing